<launch>
  <master auto="start"/>
  <node pkg="executive_trex_pr2" type="stub_ros_container" respawn="false" />
  <param name="/trex/input_file" value="sim.master.cfg"/>
  <param name="/trex/path" value="$(find executive_trex_pr2)/cfg"/>
  <param name="/trex/time_limit" value="3600"/>
  <param name="/trex/update_rate" value="1" />
  <param name="/trex/log_dir" value="$(find executive_trex_pr2)/logs"/>
  <param name="/trex/play_back" value="0"/>
</launch>
